app/models/miq_server.rb

```diff
@@ -223,6 +223,8 @@ def self.start
       ].each { |k| server_hash[k] = nil }
     end
 
+    server_hash[:has_vix_disk_lib] = server.is_vix_disk_supported?
```

Comment:
cc @agrare 

Don't think we can do our non-initializer idea with the prepend, because something still has to happen at boot